README.md

```diff
@@ -1,7 +1,35 @@
+# Settings View [![Build Status](https://travis-ci.org/atom/settings-view.svg?branch=master)](https://travis-ci.org/atom/settings-view) [![Dependency Status](https://david-dm.org/atom/settings-view.svg)](https://david-dm.org/atom/atom)
 
-Edit config settings, install packages, and change themes in Atom.
+Edit core config settings, install and configure packages, and change themes all from within Atom.
 
-Open by pressing `cmd-,` or selecting the _Atom > Preferences..._ menu.
+![Settings View](https://cloud.githubusercontent.com/assets/1476/5624241/d29ffbe6-9516-11e4-8bfe-e91623977bc7.png)
 
-![screen shot 2015-01-05 at 8 08 18 pm](https://cloud.githubusercontent.com/assets/1476/5624241/d29ffbe6-9516-11e4-8bfe-e91623977bc7.png)
+## Usage
+You can open the Settings View by navigating to _Atom > Preferences_ (Mac), _File > Preferences_ (Windows), or _Edit > Preferences_ (Linux).
+
+In order to install new packages and themes, click on the _Install_ section on the left-hand side.
+Once installed, community packages/themes (and their settings) are housed within their respective section.
+All packages/themes that have updates will be listed under the _Updates_ section.  Finally, all keybindings (including ones that community packages have added) are available in the _Keybindings_ section.
+
+Want to learn more?  Check out the [Getting Started: Atom Basics](https://atom.io/docs/latest/getting-started-atom-basics#settings-and-preferences) and [Using Atom: Atom Packages](https://atom.io/docs/latest/using-atom-atom-packages) sections in the flight manual.
+
+### Commands
+* `settings-view:open` - Opens the Settings View
+* `settings-view:show-keybindings` - Opens the _Keybindings_ section of the Settings View
+* `settings-view:uninstall-packages` - Opens the _Packages_ section of the Settings View
+* `settings-view:change-themes` - Opens the _Themes_ section of the Settings View
+* `settings-view:uninstall-themes` - Opens the _Themes_ section of the Settings View
+* `settings-view:check-for-updates` - Opens the _Updates_ section of the Settings View
+* `settings-view:install-packages-and-themes` - Opens the _Install_ section of the Settings View
+
+### Keybindings
+|Command|Windows|Mac|Linux|
+|-------|-------|---|-----|
+|`settings-view:open`|<kbd>Ctrl-,</kbd>|<kbd>⌘-,</kbd>|<kbd>Ctrl-,</kbd>|
+Custom keybindings can easily be added by referencing the above commands.
+
+### Contributing
+Always feel free to help out!  Whether it's [filing bugs and feature requests](https://github.com/atom/settings-view/issues/new) or working on some of the [open issues](https://github.com/atom/settings-view/issues), Atom's [contributing guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md) will help get you started while the [guide for contributing to packages](https://github.com/atom/atom/blob/master/docs/contributing-to-packages.md) has some extra information.
```

Comment:
> If we're adding this, should we :fire: CONTRIBUTING.md?

The benefit of having CONTRIBUTING.md is that it shows up as a [banner at the top of the page](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) when you create issues and pull requests. I'm pretty sure most folks won't click that link, but for those 1% that will -- it still might be helpful. :smile: I think we should keep it here.